docs/deployment.md

```diff
@@ -0,0 +1,16 @@
+# Deployment From PR to Production
+
+1. Create a PR. Get it reviewed.
+2. **Make sure master is merged into the PR branch** and make sure all CI tests pass.
```

Comment:
I get that you're documenting a process but on my team we prefer to rebase PRs on master to avoid merge commits all over the place. It forces the PR author to clean up the code for their PR to match changes necessary for master.

Reply:
I used to be pretty hardline on this myself, but I was amazed that no one at GitHub seemed to care ¯\_(ツ)_/¯ - I'm trying the koolaid myself right now... kinda nice.